TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -91,9 +95,12 @@ void test_multi_ticker(void)
     }
 
     Thread::wait(MULTI_TICKER_TIME_MS + TICKER_COUNT + extra_wait);
+    multi_counter_snapshot = multi_counter;
+
     for (int i = 0; i < TICKER_COUNT; i++) {
         ticker[i].detach();
     }
+    TEST_ASSERT_EQUAL(TICKER_COUNT, multi_counter_snapshot);
     TEST_ASSERT_EQUAL(TICKER_COUNT, multi_counter);
```

Comment:
So if `multi_counter_snapshot`  catches real test status, why we still check `multi_counter` against TICKER_COUNT?

Reply:
Because `detach` calls `schedule_interrupt` in some circumstances (e.g. when head event is removed), so it's good to check if no more callbacks is raised during detaching.

BTW, now I see that the first `ASSERT` can be moved up, where the snapshot is taken
And will add some comment to second check